pkg/cloud/rgraph/rnode/generic_funcs.go

```diff
@@ -243,6 +243,7 @@ func (f *UpdateFuncs[GA, Alpha, Beta]) Do(
 			if fv, err := fingerprintField(reflect.ValueOf(raw)); err != nil {
 				return err
 			} else {
+				klog.Infof("Set fingerprint to %s", fingerprint)
```

Comment:
why only for GA?

Reply:
The requirement is that every version (Alpha and Beta) should be convertible to GA. Mutable resources in postAccess copier all fields to all versions so fingerprint should be copied to GA always. 
This should be covered in test I will add TODO for that

Reply:
this was for debugging purposes. I will remove this comment.

Reply:
fixed